lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/widgets/guest_event_detail_dashboard.dart

```diff
@@ -27,6 +28,10 @@ class GuestEventDetailDashboard extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final colorScheme = Theme.of(context).colorScheme;
+    final invitationUrl =
+        context.watch<GenerateEventInvitationUrlBloc>().state.whenOrNull(
```

Comment:
reduce using watch on top of build, cause another widget to rebuild, instead wrapping BlocBuilder out of EventDashboardItem